tests/smoke/smoke_test.go

```diff
@@ -0,0 +1,13 @@
+package smoke_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Smoke", func() {
+	It("adds two numbers", func() {
+		sum := 2 + 3
```

Comment:
Hey, what exactly are we using this smoke test for?

Reply:
it's just a sample test so that this ci worflow pass all the cases, in the following pr we will remove this test case and introduce appropriate test. [https://github.com/accuknox/discovery-engine/pull/647](https://github.com/accuknox/discovery-engine/pull/647) this ci workflow is introduced for the attached pr

Reply:
Okay, then it's LGTM!